src/Repositories/Eloquent/OrderRepository.php

```diff
@@ -0,0 +1,42 @@
+<?php
+
+namespace IndieHD\Velkart\Repositories\Eloquent;
+
+use Illuminate\Database\DatabaseManager;
+use Illuminate\Database\Eloquent\Model;
+use IndieHD\Velkart\Contracts\OrderRepositoryContract;
+use IndieHD\Velkart\Models\Eloquent\Order;
+
+class OrderRepository extends BaseRepository implements OrderRepositoryContract
+{
+    /**
+     * @var Order
+     */
+    protected $order;
+
+    /**
+     * @var DatabaseManager
+     */
+    protected $db;
+
+    /**
+     * OrderRepository constructor.
+     * @param Order $order
+     * @param DatabaseManager $db
+     */
+    public function __construct(Order $order, DatabaseManager $db)
+    {
+        $this->order = $order;
+        $this->db = $db;
```

Comment:
`DatabaseManager` is unused. You should consider removing it until it is required to pass a test.